benches/helpers/miri_helper.rs

```diff
@@ -50,7 +50,7 @@ pub fn run(filename: &str, bencher: &mut Bencher) {
         let entry_def_id = tcx.hir.local_def_id(entry_node_id);
 
         bencher.borrow_mut().iter(|| {
-            eval_main(tcx, entry_def_id, None);
+            eval_main(tcx, entry_def_id, false);
```

Comment:
I'm not 100% confident in this, is this correct?

Reply:
This is `start_fn`, right?

I guess it could go either way, depending on the benchmark. But it likely makes more sense to not run with `start_fn` usually because that would dominate the benchmark.

Can we make this per-bench configurable?